### PR TITLE
added some python codes

### PR DESCRIPTION
**Pull request by sagarbabalsure**
codes for packages like numpy,random and pyqrcode

### CODE REVIEW

**Review by alnouirah — CHANGES_REQUESTED** [review]
it would be great if you can add  some extra explanation for this package !

File: Modules/random_number_en.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+while count>0:
+	y=int(input("Enter your guess: "))
+	count=count-1
+	if x==y:
+		print("Oh..your guess is correct")
+	else:
+		print("you have %d guesses remaining" %(count))
+		if count==0:
+			print("sorry you lose")
+
```
Comment (ammore5) [inline Modules/random_number_en.py]: This example is like guess game.Where user try to guess the randomly generated number in three trials.

File: Modules/qr_en.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+#By scanning this qrcode you can go to that url  
+
+
+
+
+import pyqrcode
+from pyqrcode import QRCode
+s=raw_input("Enter any url to make qr code: ")
+url=pyqrcode.create(s)
+url.svg("myqr.svg",scale=8)
```
Comment (ammore5) [inline Modules/qr_en.py]: adding new line at the end .

**Review by alnouirah — COMMENTED** [review]
Qr generator.

**Review by alnouirah — APPROVED** [review]

**Review by alnouirah — APPROVED** [review]

**Pull request merged**